src/scripts/components/VideoList.js

```diff
@@ -49,7 +50,7 @@ class VideoList extends Component<Props, void> {
       ''
     )
     return (
-      <Card margin="0 25px 0 0" nopadding footer={footer}>
+      <Card {...this.props} nopadding footer={footer}>
```

Comment:
What props are being passed here? I think it is preferable to not just pass through all props because it is not entirely clear to others what is being used/what is just extra. Can you pass explicit props?

Reply:
> What props are being passed here?

All (and only) `Cards` props. 

Do we need to pass even if we don't use not of them?
Example:
No props are used on Home
https://github.com/Paratii-Video/paratii-portal/blob/dev/src/scripts/components/pages/Home.js#L9

Using marginLeft at VideoManagerContainer
https://github.com/Paratii-Video/paratii-portal/blob/uploader-ui/src/scripts/containers/VideoManagerContainer.js#L54

Reply:
I'm not exactly sure what you mean. All I am saying here is that instead of `{...this.props}` we should do `<Component prop1={value} prop2={value2} />`, otherwise other devs cannot see what the props are w/o inspecting the code at runtime.

And if there are props not being used for sure you do not need to pass them. That is always true for any component.